get-started/part3.md

```diff
@@ -103,6 +103,19 @@ This `docker-compose.yml` file tells Docker to do the following:
 - Define the `webnet` network with the default settings (which is a
   load-balanced overlay network).
 
+Notice that we've set the Compose file version to version 3. This essentially
+makes it [swarm mode](/engine/swarm/index.md) compatible. We can make use of the
```

Comment:
Well, I think @johndmulhausen was trying to limit the amount of complexity/detail in the tutorial for a quick, task-based experience. I'm trying to walk a line here between giving more explanation with regard to the questions about the differences between `docker stack deploy` and `docker-compose up` and file names, without getting overly detailed. 

The `deploy` key with the ability to set a  swarm mode configuration was added in Compose v3, along with support for the `docker stack deploy`command. Taken together, this gives you the ability to create a swarm with a Compose file. I'm not sure that the fact that Compose v3 uses the swarm-enabled Compose API needs to be exposed here, as it's even another layer down, but I can add it if you think the current explanation is misleading in some way.

Any thoughts @ManoMarks ? @shin- @dnephin